index.js

```diff
@@ -8,10 +8,11 @@ var appName = require(path.join(process.cwd(), 'package.json')).name;
 var eyes = new Eyes();
 
 var eyesOpen = false;
-var hooked = browser.get;
+var enableSnapshotAtBrowserGet = true;
```

Comment:
This becomes a spaghetti code
to mitigate it - can you move the `bowser.get` into the `eyesWith` function? then you can also move the `enableSnapshotAtBrowserGet` to that function instead of holding it as a global variable